com.github.wwmm.PulseEffects.json

```diff
@@ -0,0 +1,267 @@
+{
+  "app-id": "com.github.wwmm.PulseEffects",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.gfortran-62"],
+  "command": "pulseeffects",
+  "rename-desktop-file": "pulseeffects.desktop",
+  "rename-icon": "pulseeffects",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--own-name=com.github.wwmm.pulseeffects",
+    "--persist=.config/PulseEffects",
```

Comment:
When you start the application does it automatically load any profiles or must the user open a dialog to select it?